docs/DeveloperGuide.md

```diff
@@ -217,88 +217,76 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **MSS**
 
-1.  User requests to add an expense.
-2.  ExpenseExpert adds the expense to the list.
+1. User requests to add an expense.
+2. User supplies the expense details.
+3. ExpenseExpert adds the expense to the list.
 
     Use case ends.
 
 **Extensions**
 
-    1a. Invalid syntax entered.
-        1a1. ExpenseExpert requests to check command parsed and enter command again.
+    2a. Expense Expert detects an error in details parsed.
+        2a1. ExpenseExpert requests to check the details of expense to be added and enter command again.
```

Comment:
Should it be "... prompts user to check the details of expense to be added and enter command again"?